firebase-ai/src/test/java/com/google/firebase/ai/common/util/descriptorToJson.kt

```diff
@@ -144,8 +144,8 @@ internal fun typeNameFromKind(kind: SerialKind): String {
  * Our serialization classes are nested within the public-facing classes, and that's the name we
  * want in the json output. There are two class names
  *
- * - `com.google.firebase.ai.type.Content.Internal` for regular scenarios
- * - `com.google.firebase.ai.type.Content.Internal.SomeClass` for nested classes in the
+ * - `com.google.firebase.vertexai.type.Content.Internal` for regular scenarios
+ * - `com.google.firebase.vertexai.type.Content.Internal.SomeClass` for nested classes in the
```

Comment:
You might have accidentally reverted this change. It should be `ai`.
```suggestion
 * - `com.google.firebase.ai.type.Content.Internal` for regular scenarios
 * - `com.google.firebase.ai.type.Content.Internal.SomeClass` for nested classes in the
```